parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1327,7 +1341,8 @@ loop = do
 
           allTypes :: Map Reference (Type v Ann) <-
             fmap Map.fromList . for (toList neededTypes) $ \r ->
-              (r,) . fromJust <$> (eval . LoadTypeOfTerm) r
+              (r,) . fromMaybe (Type.builtin External "unknown type")
+              <$> (eval . LoadTypeOfTerm) r
```

Comment:
I removed all `fromJust` calls from the codebase as part of this PR (original but was a fromJust runtime crash). 

This treatment here is a little dubious (it'll get a type of "unknown type"), but should only get hit if codebase is corrupted somehow.

Reply:
Better than crashing 👍